discord/member.py

```diff
@@ -464,18 +470,19 @@ def _presence_update(self, data: PartialPresenceUpdate, user: UserPayload) -> Op
 
     def _update_inner_user(self, user: UserPayload) -> Optional[Tuple[User, User]]:
         u = self._user
-        original = (u.name, u.discriminator, u._avatar, u.global_name, u._public_flags)
+        original = (u.name, u.discriminator, u._avatar, u.global_name, u._public_flags, u._avatar_decoration_data)
```

Comment:
dict comparisons like this are a bit iffy, we should probably compare a key in the dict instead

Reply:
It's checked with the `sku_id` now and if it differs from the original, the new dict is assigned.